gulpfile.js

```diff
@@ -240,25 +242,26 @@ async function generateReactWebviewsBundle() {
 			typecheckPlugin()
 		],
 		sourcemap: prod ? false : 'inline',
-		metafile: !prod,
+		metafile: true,
```

Comment:
I am always creating a metafile now. This file won't be included in the vsix file as it present in .vscodeignore.